### DISCUSSION TITLE
Scripts->Suggestions

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/mympd-scripts#21] [General]
Hi JC,

Very exciting for the new features you provide day by day! :)

Two small suggestions:

- When someone "Import" a script from the list you have made, it should remove that script from this list. This will help, someone like me, because i choose 6-7 scripts at the same time and when i go back to the list i do not remember everything i choose!
- Second, i cannot find Youtube script in this list. Maybe i cannot see it, though! 

Thank you!

**Comment by jcorporation** [comment]
1) Good idea, greying already imported scripts out should be the best.
2) The script is not finished and finally ported to the new scripting api. I hope @sevmonster get time to finalize it. It is currently in a feature branch and waits.

**Reply by Emilot** [reply]
JC all these scripts need MPD 0.24 or some of these?
All of these need an account to MusicBrainz and need to tag our music library with MusicBrainz (Picard)? The last part of the question is for TagArt and AlbumArt scripts.

Thanx!

**Reply by jcorporation** [reply]
The README.md files in each script directory, which are also linked in the script list should describe the dependencies.

**Reply by Emilot** [reply]
> The README.md files in each script directory, which are also linked in the script list should describe the dependencies.

Ok! Then yes

Thanx
